src/core-skills/bmad-brainstorming/scripts/tests/test_memlog.py

```
@@ -0,0 +1,237 @@
+    return [ln for ln in body_of(ws).splitlines() if ln.startswith("- ")]
+
+
+def init(ws, **fields):
+    fields = fields or {"topic": "Reinvent the lunchbox", "goal": "ideas for a pitch"}
+    argv = ["init", "--workspace", ws]
+    for k, v in fields.items():
+        argv += ["--field", f"{k}={v}"]
+    assert memlog.main(argv) == 0
+
+
+def append(ws, text, type=None, by=None):
```

Ruff (0.15.14)

[error] Function argument `type` is shadowing a Python builtin

(A002)